deps/curl.mk

```diff
@@ -0,0 +1,46 @@
+## CURL ##
+
+CURL_SRC_TARGET := $(BUILDDIR)/curl-$(CURL_VER)/.libs/libcurl.$(SHLIB_EXT)
+CURL_OBJ_TARGET := $(build_shlibdir)/libcurl.$(SHLIB_EXT)
+
+ifneq ($(OS),WINNT)
+CURL_LDFLAGS := "-Wl,-rpath,'$(build_libdir)'"
```

Comment:
I don't think this is enough, ref https://github.com/JuliaLang/julia/pull/17458/files